Presentation.Web/app/services/userServices.ts

```diff
@@ -185,6 +185,20 @@
             return this.$http.get<Kitos.API.Models.IApiWrapper<any>>("api/authorize/GetOrganizations");
         };
 
+        /**
+         * returns the organizations for the user whos credentials have been authorized
+         * @param orderByPropertyName - orders the list by property with given name
+         * @param orderByAsc - should the list be in ascending or descending order
+         */
+        getOrganizationsOrderedByProperty = (orderByPropertyName: string, orderByAsc?: boolean) => {
+            let query = `api/authorize/GetOrganizations?orderBy=${orderByPropertyName ?? 'Name'}`;
```

Comment:
ahh - saw that your orderByPropertyName was no optional. In that regard remove the ?? 'Name' and introduce a parameter check in stead because missing that argument will be a programmers error (exception) wheras the ordering parameter is optional and is now treated as such 👍